README.md

```diff
@@ -44,6 +48,7 @@ Veneur expects to have a config file supplied via `-f PATH`. The include `exampl
 
 * `api_hostname` - The Datadog API URL to post to. Probably `https://app.datadoghq.com`.
 * `metric_max_length` - How big a buffer to allocate for incoming metric lengths. Metrics longer than this will get truncated!
+* `flush_max_per_body` - how many metrics to include in each JSON body POSTed to Datadog. Veneur will POST multiple bodies in parallel if it goes over this limit. A value around 100k-150k is recommended; in practice we've seen Datadog reject bodies over about 195k.
```

Comment:
Update! I did some testing in QA with a script that generated about 185k metric names per flush and ran it with old veneur, and new veneur at 3 flush sizes (5k, 10k and 100k). Here's the HTTP total time post-waitgroup:

<img width="546" alt="screenshot 2016-06-14 07 47 43" src="https://cloud.githubusercontent.com/assets/45159/16047441/e6932a42-3204-11e6-9f86-287ee9e9c8f3.png">

5k seems like the best option and has no perceivable impact on metric processing.